### PR TITLE
Fix overriding binary install 

### PR DESCRIPTION
**Pull request by abhijelly**
### Description

Referred the examples and docs suggested in #1928

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extend the tutorials / documentation [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Document API changes relevant to the user in the [MIGRATION.md](https://github.com/ros-planning/moveit2/blob/main/MIGRATION.md) notes
- [ ] Create tests, which fail without this PR [reference](https://moveit.picknik.ai/humble/doc/examples/tests/tests_tutorial.html)
- [ ] Include a screenshot if changing a GUI
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/ros-planning/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: moveit_py/src/moveit/moveit_py_utils/CMakeLists.txt [hunk 1]
```diff
@@ -13,4 +13,4 @@ install(
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin)
 
-install(DIRECTORY include/ DESTINATION include)
+install(DIRECTORY include/ DESTINATION include/moveit_py)
```
Comment (tylerjw) [inline moveit_py/src/moveit/moveit_py_utils/CMakeLists.txt:16]: As the package name is `moveit_py_utils`, that should also be the install directory?

File: moveit_planners/test_configs/prbt_ikfast_manipulator_plugin/CMakeLists.txt [hunk 2]
```diff
@@ -61,7 +61,7 @@ install(
   LIBRARY DESTINATION lib
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin
-  INCLUDES DESTINATION include
+  INCLUDES DESTINATION include/prbt_ikfast_manipulator_plugin
```
Comment (tylerjw) [inline moveit_planners/test_configs/prbt_ikfast_manipulator_plugin/CMakeLists.txt:64]: I belive this should be `moveit_resources_prbt_ikfast_manipulator_plugin` as that is the name of the project.

File: moveit_planners/ompl/CMakeLists.txt [hunk 3]
```diff
@@ -31,7 +31,7 @@ install(TARGETS moveit_ompl_interface moveit_ompl_planner_plugin
   LIBRARY DESTINATION lib
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin
-  INCLUDES DESTINATION include
+  INCLUDES DESTINATION include/moveit_ros_ompl
```
Comment (tylerjw) [inline moveit_planners/ompl/CMakeLists.txt:34]: Similar to my other comments, I believe this should be `moveit_planners_ompl` to match the project name.

**Comment by abhijelly** [comment]
Hi @tylerjw @ChrisThrasher - I've made the suggested changes!

**Review by tylerjw — APPROVED** [review]

File: moveit_py/src/moveit/moveit_py_utils/CMakeLists.txt [hunk 4]
```diff
@@ -13,4 +13,4 @@ install(
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin)
 
-install(DIRECTORY include/ DESTINATION include)
+install(DIRECTORY include/ DESTINATION include/moveit_py_utils)
```
Comment (tylerjw) [inline moveit_py/src/moveit/moveit_py_utils/CMakeLists.txt:16]: I'm sorry I made a mistake in what I suggested here. The package is `moveit_py`.

```suggestion
install(DIRECTORY include/ DESTINATION include/moveit_py)
```

**Review by henningkayser — APPROVED** [review]

**Pull request merged**